test/inspect-test.js

```diff
@@ -333,17 +351,21 @@ tape('v8 inspect', (t) => {
     }
   });
 
-  sess.linesUntil(/}>/, (lines) => {
-    lines = lines.join('\n');
-    t.ok(/internal fields/.test(lines), 'method.scopedAPI.internalFields');
-  });
+  if (process.version >= 'v5.0.0') {
```

Comment:
Is this safe to do?

```
> process.version
'v10.0.0-pre'
> process.version >= 'v5.0.0'
false
```

Reply:
@cjihrig Oops, Node core jenkins scripts bug dejavu..this pattern is used in other places as well, I would like to fix them in a separate PR before v10.x releases.